fabfile.py

```diff
@@ -61,6 +61,11 @@ def deploy():
 
 def migrate():
     with cd(env.code_dir):
+        run('su %s -c "git fetch && git status"' % env.unprivileged_user)
+        if confirm("Update to latest commit in this branch?", default=False):
+            run('su %s -c "git pull"' % env.unprivileged_user)
+            run('su %s -c "PATH=%s/bin:$PATH pip install -r requirements.txt"' % (env.unprivileged_user, env.venv_dir))
+            run('sudo systemctl restart openoversight')
```

Comment:
we probably don't want to restart until after the migrations have run.

Reply:
Addressed in 1a6ad9d